content/2021-losangeles/diversity.md

```diff
@@ -19,7 +19,9 @@ from the PromCon Online 2021 on May 3, 2021.
 
 To encourage and increase the participation of a diverse community, we are pleased to offer diversity and need-based registration scholarships, discounts, and travel funding to our conferences.
 
-[**APPLY NOW**](https://www.surveymonkey.com/r/Co-LoScholarshipNA21)
+<a class="btn btn-lg btn-default" href="https://www.surveymonkey.com/r/Co-LoScholarshipNA21" target="_blank" role="button">
+  <i class="fa fa-pencin"></i>&nbsp;&nbsp;Apply now
```

Comment:
I guess this should be:

```suggestion
  <i class="fa fa-pencil"></i>&nbsp;&nbsp;Apply now
```

...at least currently the pencil icon doesn't show up.